invoice2data/main.py

```diff
@@ -90,7 +93,15 @@ def main():
                     date=res['date'].strftime('%Y-%m-%d'),
                     desc=res['desc'])
                 shutil.copyfile(f.name, join(args.copy, filename))
-    invoices_to_csv(output, 'invoices-output.csv')
+    if args.change_csv_name:
```

Comment:
Instead of this long `if`-condition it may be possible to set a default value directly at `parser.add_argument`?

See here for many examples: https://docs.python.org/3/library/argparse.html

Reply:
@duskybomb ^^

Reply:
Done

Reply:
Looks better. I still wanna make some changes. Will merge later.